lib/zexbox/flags.ex

```diff
@@ -124,6 +124,10 @@ defmodule Zexbox.Flags do
   defp parse_config(config) do
     config
     |> Map.delete(:sdk_key)
-    |> Map.update(:file_paths, [], fn paths -> Enum.map(paths, &String.to_charlist/1) end)
```

Comment:
We don't need this anymore this can now be a binary list in this version of the client.